### DISCUSSION TITLE
Forking GTK+ 3

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#688] [Ideas]
I agree GTK 3 is a very good toolkit and it's the best available. It has nice, powerful themeing, good layout features, clean bindings, touch support... It's a pleasure to work with, and I wouldn't want to switch to something else, nor do you have any plans to.

But the GTK 3 has some missing features: its file browser is very bad, it should be configurable; it will be unsupported and is considered outdated. Are you going to fork it?

**Comment by Secret-chest** [comment]
To see why GTK 4 is a bad choice, look at the new Inkscape menus which had to be used because GTK 4 doesn't support real menus. No nesting (!) - it just replaces the main menu, and much more.

**Reply by JosephMcc** [reply]
What are you talking about? You mean submenus? Menu in GTK4
![Screenshot from 2024-11-29 10-09-06](https://github.com/user-attachments/assets/af68fc53-871c-4879-ad14-953c8e766420)

**Reply by Secret-chest** [reply]
What app is this? Because in Inkscape 1.4, when I click on a submenu, it replaces the main one.

**Reply by Secret-chest** [reply]
Is the font being changed to Inter?

**Comment by Secret-chest** [comment]
Indeed, AWF GTK 4 has submenus.
